libfuse/fs.go

```diff
@@ -89,19 +89,20 @@ func (f *FS) processNotifications(ctx context.Context) {
 			}
 			return
 		case i := <-f.notifications.Out():
-			notifyFn, ok := i.(func())
-			if !ok {
-				f.log.CWarningf(ctx, "Got a bad notification function: %v", i)
-				continue
-			}
-			notifyFn()
-			f.notificationGroup.Done()
+			func() {
```

Comment:
libdokan has these files too, so may as well fix those.

I have a branch somewhere that unifies the two -- I can send that out after this one.